src/hoverboard-app.html

```diff
@@ -34,7 +35,7 @@
       :host {
         display: block;
         position: relative;
-        min-height: 100vh;
+        min-height: 200vh;
```

Comment:
We can add such values to test on local environments but never push to the repository